src/js/smooch.jsx

```diff
@@ -232,4 +232,19 @@ export class Smooch {
     showSettingsNotification() {
         store.dispatch(showSettingsNotification());
     }
+
+    getConversation() {
+        const conversation = store.getState().conversation;
+        const convoNotFoundError = () => {
+            throw new Error('conversation not found');
+        };
+
+        if (conversation) {
```

Comment:
The use case is:

* No conversation started
* Your backend posts the first message through the REST API
* Call `getConversation()` to tell the SDK that you have started a conversation

it needs to fetch because there is no local copy of the conversation yet